components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Extensions/PublishedNodesEntryModelEx.cs

```diff
@@ -28,7 +28,7 @@ public static class PublishedNodesEntryModelEx {
                 return false;
             }
 
-            if (string.Compare(model.EndpointUrl.OriginalString, that.EndpointUrl.OriginalString, StringComparison.OrdinalIgnoreCase) != 0) {
+            if (model.EndpointUrl != that.EndpointUrl) {
```

Comment:
Why make comparison case-sensitive ? URL comparison should in general be case-insensitive.

Reply:
I'm comparing now the **Uri** EndpointUrl objects no longer the strings.

Reply:
Ah, right. Would this also ignore trailing `/` characters in URIs ?

Reply:
yes, trailing `/` are ignored, among others...